src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,133 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
+
+    @BeforeEach
+    void setUp() {
+    }
+
+    @Test
+    public void user_register_with_short_login_notOK() {
+        User userWithShortLogin = new User();
+        userWithShortLogin.setPassword("123456");
+        userWithShortLogin.setId(1234321L);
+        userWithShortLogin.setLogin("231");
+        userWithShortLogin.setAge(20);
+
+        RegistrationException exception = assertThrows(RegistrationException.class, () -> {
+            registrationService.register(userWithShortLogin);
+        });
+
+        assertEquals("This login is to short,  min 6 characters", exception.getMessage());
+
+    }
+
+    @Test
+    public void user_register_with_short_passwors_notOK() {
```

Comment:
better naming suggestion
```suggestion
    public void userRegister_shortPassword_notOk() {
```